addons/mail/static/src/components/composer_text_input/composer_text_input.js

```diff
@@ -165,11 +165,13 @@ class ComposerTextInput extends Component {
         }
         this._textareaRef.el.value = this.composer.textInputContent;
         this._mirroredTextareaRef.el.value = this.composer.textInputContent;
-        this._textareaRef.el.setSelectionRange(
-            this.composer.textInputCursorStart,
-            this.composer.textInputCursorEnd,
-            this.composer.textInputSelectionDirection,
-        );
+        if (this.composer.hasFocus) {
```

Comment:
I would be more comfortable with condition on actual DOM focus for now, but this is acceptable enough, as I didn't notice any issue.

As I can't test safari I can't help much more, feel free to handle as you see fit.